nconnect.go

```diff
@@ -311,29 +316,38 @@ func (nc *nconnect) StartClient() error {
 
 	remoteTunnelAddr := nc.opts.RemoteTunnelAddr
```

Comment:
`remoteTunnelAddr` is now a pointer to original opts. It's better to make a copy